DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -6,14 +6,14 @@
 @inject IConfiguration Configuration
 
 @{
-  var headerExtension = (string)ViewData[LayoutViewDataKeys.Application];
-  var shouldDisplayHelpMenuItem = headerExtension != "Tracking System";
   var dlsSubApplication = (DlsSubApplication)ViewData[LayoutViewDataKeys.DlsSubApplication];
   if (dlsSubApplication != null) {
     ViewData[LayoutViewDataKeys.Application] ??= dlsSubApplication.HeaderExtension;
     ViewData[LayoutViewDataKeys.HeaderPath] ??= dlsSubApplication.HeaderPath;
     ViewData[LayoutViewDataKeys.HeaderPathName] ??= dlsSubApplication.HeaderPathName;
   }
+  var headerExtension = (string)ViewData[LayoutViewDataKeys.Application];
+  var shouldDisplayHelpMenuItem = headerExtension != "Tracking System";

```

Comment:
Oh, good thing you moved this, else I'd have missed it. This should be updated to use the `dlsSubApplication` if `ViewData["Application"]` is null.

Reply:
Oh sorry! Yes you're setting it above, so you're all good. Never mind.

Reply:
Actually @AlexJacksonDS This detail of whether the help menu item should be displayed seems to be determined by the application - if so it could just live on the DlsSubApplication class.

Reply:
Yeah that's fair, probably a better way of doing it, although we're going to be in an awkward middle ground with all the tracking system pages, unless we update them all to the new system, or go around adding a new VIewData value everywhere

Reply:
Yeah, so we can do what we're doing with other view data properties on Layout, and use the old method if ViewData[DlsSubApplication] == null. That saves us having to re-test all of the tracking system.